lib/utils/messages.ts

```diff
@@ -71,17 +71,18 @@ export const dockerignoreHelp = `\
 DOCKERIGNORE AND GITIGNORE FILES  
 By default, the balena CLI will use a single ".dockerignore" file (if any) at
 the project root (--source directory) in order to decide which source files to
-exclude from the "build context" (tar stream) sent to balenaCloud, Docker daemon
-or balenaEngine. In a microservices (multicontainer) application, the source
-directory is the directory that contains the "docker-compose.yml" file.
+exclude from the "build context" (tar stream) sent to balenaCloud, Docker
+daemon or balenaEngine. In a microservices (multicontainer) application, the
+source directory is the directory that contains the "docker-compose.yml" file.
```

Comment:
One of the commits focuses on ensuring that the output of `balena help push` fits in 80 characters. (It would be nice to add some test code to check this at some point.)

Reply:
Yeah that would be nice.